src/Microsoft.AspNetCore.Mvc.Formatters.Json/Internal/MvcJsonLoggerExtensions.cs

```diff
@@ -8,13 +8,13 @@ namespace Microsoft.AspNetCore.Mvc.Formatters.Json.Internal
 {
     internal static class MvcJsonLoggerExtensions
     {
-        private static readonly Action<ILogger, string, Exception> _jsonInputFormatterCrashed;
+        private static readonly Action<ILogger, Exception> _jsonInputFormatterCrashed;
 
         private static readonly Action<ILogger, string, Exception> _jsonResultExecuting;
 
         static MvcJsonLoggerExtensions()
         {
-            _jsonInputFormatterCrashed = LoggerMessage.Define<string>(
+            _jsonInputFormatterCrashed = LoggerMessage.Define(
                 LogLevel.Debug,
```

Comment:
No, this is correct as-is, this is a failure to parse the request body. The error will *generally* be surfaced to the user as a 400